lib/generators/flipper/setup_generator.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails/generators/active_record'
+
+module Flipper
+  module Generators
+    class SetupGenerator < ::Rails::Generators::Base
+      desc 'Peform any necessary steps to install Flipper'
+
+      class_option :token, type: :string, default: nil, aliases: '-t',
+        desc: "Your Flipper Cloud development token"
+
+      def generate_active_record
+        invoke 'flipper:active_record' if defined?(Flipper::Adapters::ActiveRecord)
```

Comment:
Any specific reason for not using `generate` here instead of `invoke`? And on the super-minor side, using the conditional as a guard clause instead?

Reply:
The tests fail when I try to use `generate` since they're not running in the context of a real Rails app:

```
/Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/3.3.0+0/bundled_gems.rb:68:in `require': cannot load such file -- /Users/bkeepers/projects/flipper/config/boot (LoadError)
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/3.3.0+0/bundled_gems.rb:68:in `block (2 levels) in replace_require'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/gems/3.3.0+0/gems/railties-7.0.8/lib/rails/app_loader.rb:58:in `block in exec_app'
	from <internal:kernel>:187:in `loop'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/gems/3.3.0+0/gems/railties-7.0.8/lib/rails/app_loader.rb:48:in `exec_app'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/gems/3.3.0+0/gems/railties-7.0.8/lib/rails/cli.rb:7:in `<top (required)>'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/3.3.0+0/bundled_gems.rb:68:in `require'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/3.3.0+0/bundled_gems.rb:68:in `block (2 levels) in replace_require'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/lib/ruby/gems/3.3.0+0/gems/railties-7.0.8/exe/rails:10:in `<top (required)>'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/bin/rails:25:in `load'
	from /Users/bkeepers/.asdf/installs/ruby/3.3.0-rc1/bin/rails:25:in `<main>'
```